package.json

```diff
@@ -23,8 +23,8 @@
     "chai": "4.1.2",
     "eslint": "4.19.1",
     "eslint-config-google": "0.9.1",
-    "lerna": "3.10.2",
-    "mocha": "5.2.0",
+    "lerna": "^3.22.1",
```

Comment:
can remove the ^ so it doesnt update without us knowing

Reply:
already remove kindly checked.